02-spatial-data.Rmd

```diff
@@ -296,29 +314,28 @@ They could be easily describes as a well-known text:
 plot(st_as_sfc(c("POINT(5 2)")), axes = TRUE, main = "POINT")
 ```
 
-A linestring is represented by a sequence of points with linear interpolation between points, for example:
+A linestring is a sequence of points with a straight line connecting the points, for example:
 
 - `LINESTRING (1 5, 4 4, 4 1, 2 2, 3 2)`
 
 ```{r linestring, echo=FALSE}
 plot(st_as_sfc("LINESTRING(1 5, 4 4, 4 1, 2 2, 3 2)"), axes = TRUE, main = "LINESTRING")
 ```
 
-Linestring cannot have self intersecting line part.
```

Comment:
Again not my work but agree with the comment and suggest the self-intersecting part to keep it simple. Please make the necessary change to do this.